src/starkware/crypto/ffi/js/CMakeLists.txt

```diff
@@ -1,14 +1,14 @@
 add_custom_command(
-    OUTPUT ${CMAKE_CURRENT_SOURCE_DIR}/libcrypto_c_exports.so
+    OUTPUT ${CMAKE_CURRENT_SOURCE_DIR}/libcrypto
     COMMAND
         ${CMAKE_COMMAND} -E copy
-        ${CMAKE_CURRENT_BINARY_DIR}/../libcrypto_c_exports.so
-        ${CMAKE_CURRENT_SOURCE_DIR}/libcrypto_c_exports.so
+        ${CMAKE_CURRENT_BINARY_DIR}/../libcrypto_c_exports.*
```

Comment:
try:
libcrypto_c_exports${CMAKE_SHARED_LIBRARY_SUFFIX}

Also for the OUTPUT and DEPENDS line